plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -123,11 +125,13 @@
             .add("org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe")
             .add("parquet.hive.serde.ParquetHiveSerDe")
             .build();
+    private static final int DEFAULT_DOMAIN_COMPACTION_THRESHOLD = 32;
```

Comment:
We should not have this hard-coded constant

Reply:
done.